tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -548,6 +552,11 @@ public void EnableTransport(TestTransports transport)
             }
         }
 
+        public void EnableDataPipeline(bool isEnabled)
```

Comment:
nit, but I'd be inclined to not add this and just to set the env vars directly in the test. That's what we do in general, and I think it makes things more clear in the test 🙂